src/pages/Enter/Enter.scss

```diff
@@ -0,0 +1,31 @@
+@use "../../styles/variables" as *;
+.enter {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  height: 100vh;
```

Comment:
I'd recommend using 100% here, as the layout component already has 100vh on it

Reply:
done